packages/edit-site/src/components/layout/index.js

```diff
@@ -236,8 +235,6 @@ export default function Layout( { route } ) {
 						</div>
 					) }
 				</div>
-
-				<SavePanel />
```

Comment:
This PR probably removes the save panel from pages that don't render an editor, I think that's probably a bad idea. We should be able to "save" from these pages as well.

Reply:
see: https://github.com/WordPress/gutenberg/pull/62863#issuecomment-2201990675